metadata/validation.json

```diff
@@ -1,32 +1,33 @@
 {
     "vendors": [
         {
-            "name": "Acme Fictional Corporation",
-            "website": "www.acme-fictional.com",
-            "emails": [
-                "acme@acme-fictional.com"
-            ],
-            "phone_numbers": [
-                { "number": "000-000-0000", "type": "voice" }
+            "name": "Acme Fictional",
+            "website": "http:fictional-acme.com",
+            "emails" : [ "acme@acme.com" ],
+            "phoneNumbers": [
+                { "number": "1-800-000-0000", "type": "voice" }
             ],
+            "address": {
+                "street1": "Beach Dr.",
+                "street2": null,
+                "street3": null,
+                "locality": "Acme Town",
+                "region": "NC",
+                "country": "USA",
+                "postalCode": "00000"
+            },
             "contacts": [
                 {
-                    "full_name": "Wyle E. Coyote",
+                    "fullName": "Jane Smith",
                     "emails": [
-                        "wcoyote@acme-fictional.com"
+                        "jane.smith@acme.acme"
```

Comment:
acme.acme ? is that supposed to be acme.com ?

Reply:
yeah that was a typo, but it worked for the creation and query